base/linalg/rowvector.jl

```diff
@@ -165,6 +165,7 @@ end
 
 # inner product -> dot product specializations
 @inline *{T<:Real}(rowvec::RowVector{T}, vec::AbstractVector{T}) = dot(parent(rowvec), vec)
+@inline *{T<:Real}(rowvec::ConjRowVector{T}, vec::AbstractVector{T}) = dot(rowvec', vec)
```

Comment:
All `ConjRowVector * AbstractVector` use `dot` on the line below (now line 169), @stevengj. 

It also seemed reasonable that ours and user-defined implementations of `dot` should match `RowVector{<:Real} * AbstractVector{<:Real}`. However we can't extend line 167 to all element types... that wouldn't be correct, which is why the generic fallback for things like `Vector{Complex{T}}.' * Vector{Complex{T}}` for `T` non-BLAS float is needed on 171/172.

(But now I'm wondering, wasn't there a BLAS-related method like `dot_u` method somewhere? Going by memory here...)